tests/Kernel/fixtures/rendering.yml

```diff
@@ -1717,6 +1759,34 @@
       'h1.ecl-card__title a': "Example title"
       'div.ecl-card__description .ecl-paragraph': "Example description"
       'a.ecl-button .ecl-button__label': "Link title"
+- array:
+    '#type': pattern
+    '#id': featured_item
+    '#variant': "default"
+    '#fields':
+      title: "Example title"
+      description: "Example description"
+      footer_items:
+        - icon: "calendar"
+          text: "15-16 November 2019, 08:00 AM"
+        - icon: "location"
+          text: "Brussels, Belgium"
+  assertions:
+    count:
+      "footer.ecl-card__footer": 1
+    equals:
+      'li.ecl-card__info-item:nth-child(1)' : '<svg class="ecl-icon ecl-icon--xs" focusable="false" aria-hidden="true"><use xlink:href="/themes/custom/oe_theme/dist/ec/images/icons/sprites/icons.svg#general--calendar"></use></svg><span class="ecl-card__info-label">15-16 November 2019, 08:00 AM</span>'
+      'li.ecl-card__info-item:nth-child(2)' : '<svg class="ecl-icon ecl-icon--xs" focusable="false" aria-hidden="true"><use xlink:href="/themes/custom/oe_theme/dist/ec/images/icons/sprites/icons.svg#general--location"></use></svg><span class="ecl-card__info-label">Brussels, Belgium</span>'
+- array:
+    '#type': pattern
+    '#id': featured_item
+    '#variant': "default"
+    '#fields':
+      title: "Example title"
+      description: "Example description"
+  assertions:
+    count:
+      "footer.ecl-card__footer": 0
```

Comment:
We could just probably add this assertion to one of the previous tests and save creating one array just for this.

Reply:
Ok I'll move it up.